en-US/privacy/index.lang

```diff
@@ -17,14 +18,33 @@ If you want to make a correction to your information, or you have any questions
 Data Privacy Principles
 
 
+## TAG: updates-052018
+# Links to https://www.mozilla.org/privacy/faq/
+;Mozilla's Data Privacy Principles inspire our practices that respect and protect people who use the Internet. Learn how these principles shape Firefox and all of our products in this <a href="%(faq)s">FAQ</a>.
+Mozilla's Data Privacy Principles inspire our practices that respect and protect people who use the Internet. Learn how these principles shape Firefox and all of our products in this <a href="%(faq)s">FAQ</a>.
+
+
+# This string is obsolete. Do not remove.
 ;Mozilla is an open source project with a mission to improve your Internet experience. This is a driving force behind our data privacy practices. <a href="%(link)s">Read More</a>
 Mozilla is an open source project with a mission to improve your Internet experience. This is a driving force behind our data privacy practices. <a href="%(link)s">Read More</a>
 
 
+## TAG: updates-052018
 ;Our Privacy Notices
```

Comment:
This is not correct, it's the other way around
https://github.com/mozilla/bedrock/pull/5850/files#diff-a8503aba570309aa9167f580d02641edL80

Reply:
Oops, gosh how did I manage to do that.